sanic/app.py

```diff
@@ -1027,8 +1027,15 @@ def run(
                 "stop_event will be removed from future versions.",
                 DeprecationWarning,
             )
-        # compatibility old access_log params
-        self.config.ACCESS_LOG = access_log
+        # if access_log is passed explicitly change config.ACCESS_LOG
+        if access_log is not None:
+            if access_log in [True, False]:
```

Comment:
@Amanit  `isinstance(True, bool)` ??

:)

Reply:
Yeah, it was a late night, thanks :)